docs/qa/test_engineering.rst

```diff
@@ -9,36 +9,39 @@ The Delivery Console is not currently being tested, as it’s a much lower prior
 The test strategy incorporates manual and automated testing. Different approaches are outlined for each area. 
 
 Normandy Server
----------------
+--------------
```

Comment:
RST warns that this udnerline is too short, compared to the text it is underlining. Our set up treats warnings as errors, so this is a problem.